CMakeLists.txt

```diff
@@ -69,6 +69,10 @@ set(BUILD_NUMBER
     ""
     CACHE STRING "Build number of the firmware. Resolved automatically if not specified."
     )
+set(CUSTOM_COMPILE_OPTIONS
```

Comment:
future work - think about a more standard/widely used solution like CFLAGS/CXXFLAGS
this is ok for now